src/main/webapp/step2.html

```diff
@@ -18,5 +18,7 @@ <h2 class="title">Choose mode of transportation</h2>
                 <a class="next-button" href="step3.html"><b>Next</b></a>
             </div>
         </div>
+
+        <button class="action" onclick="window.getTransportMethods()">Submit</button>
```

Comment:
The function name is confusing (to me), from this name I expect it to return transport methods - that contradict with it used as event listener. Please fix.

Reply:
I modified it so the name it's more meaningful